### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jsice**
# Overview
Add document for new model `Moderation` 
Please help me with this, if you have any idea, don't hesitate to give a comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,13 +56,22 @@ Datawow.text_ja
...
+#### Prediction class `Datawow.prediction`
+There is 1 API for prediction class
 
 ```ruby
 Datawow.prediction
 ```
 
+---
+
+#### Moderation class `Datawow.moderation`
```
Comment (zaoldreck) [inline README.md]: Add link postman from email support@datawow.io

File: README/moderation_docs.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  postback_url: postback_url,
+  postback_method: postback_method,
+  custom_id: custom_id,
+  info: info,
+  sandbox: sandbox
+}
+Datawow.moderation.create(params)
+```
+
+Data could be text or image or else according to your project's input type
```
Comment (aunsira) [inline README/moderation_docs.md]:
```suggestion
`data` could be text or image's url depends on your project's input type
```

File: README/moderation_docs.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+#### result
+```
+response = Datawow.moderation.create({
+  data: YOUR_DATA,
+  custom_id: 'data-1'
+})
+
+print(response.body)
+```
+If you could create moderation successfully, you will receive response like below:
```
Comment (aunsira) [inline README/moderation_docs.md]:
```suggestion
You will receive response like below, once you created moderation successfully.
```

**Comment by zaoldreck** [comment]
This branch okey for release code and add update documentation later.

**Comment by zaoldreck** [comment]
@jsice Merge

**Pull request merged**